frontend/src/components/VItemGroup/meta/VItemGroup.stories.mdx

```diff
@@ -1,34 +1,33 @@
-import { ref } from "vue"
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs"
 
+import { ref } from "vue"
 import { itemGroupDirections } from "~/types/item-group"
-
 import VItemGroup from "~/components/VItemGroup/VItemGroup.vue"
 import VItem from "~/components/VItemGroup/VItem.vue"
 import VIcon from "~/components/VIcon/VIcon.vue"
 import VPopover from "~/components/VPopover/VPopover.vue"
 import VButton from "~/components/VButton.vue"
-
 import replay from "~/assets/icons/replay.svg"
-
 import pause from "~/assets/icons/pause.svg"
 import play from "~/assets/icons/play.svg"
 import close from "~/assets/icons/close.svg"
 
-const icons = [close, pause, play, replay]
+<Meta
+  title="Components/VItemGroup"
+  components={VItemGroup}
+  argsTypes={{
```

Comment:
```suggestion
  argTypes={{
```

Reply:
Because of the extra `s` in the parameter name (arg**s**Types), these were not used in the Storybook controls.